### DISCUSSION TITLE
Need to change from the localhost to the IPaddress. Need to Externalize to outside Instead of Local Host

### DISCUSSION THREAD

**Discussion by kondetiramanakumar** [apache/camel-karavan#1255] [Q&A]
### Ask a question

<img width="946" alt="image" src="https://github.com/apache/camel-karavan/assets/14121796/3bde830e-c8ee-4f82-9a73-57bb76e15774">
I am able to run the Karavan-app in the Ubuntu Server Instance.  I want to change the host name from localhost to 192.168.120.167 . May I know which files should I need to change in the karavan sources?????

### Variant

None

### Container Management (if applicable)

None

### Operating System (if applicable)

None

### Version

None

**Comment by kondetiramanakumar** [comment]
@davsclaus Required your inputs for this????

**Comment by davsclaus** [comment]
its a quarkus question
https://quarkus.io/guides/http-reference
